### PR TITLE
DOC: clarify idxmax behaviour issue #18206

### LINKED ISSUE #18206: DataFrame.idxmax returns NaN if all entries are NaN, contradicts idxmax documentation
#### Code Sample, a copy-pastable example if possible

```python
(Pdb) df = pd.DataFrame({'a': [pd.np.nan]*10})
(Pdb) df.idxmax()
a   NaN
dtype: float64
(Pdb) df.idxmax(skipna=True)
a   NaN
dtype: float64
(Pdb) pd.__version__
'0.20.2'
```
#### Problem description

According to [DataFrame.idxmax documentation](https://pandas-docs.github.io/pandas-docs-travis/generated/pandas.DataFrame.idxmax.html): `If an entire row/column is NA, the result will be first index`

#### Expected Output
```
df.idxmax(skipna=True)
a     0
```
#### Output of ``pd.show_versions()``

<details>

------------------
commit: None
python: 3.4.2.final.0
python-bits: 64
OS: Linux
OS-release: 4.9.0-4-amd64
machine: x86_64
processor: 
byteorder: little
LC_ALL: None
LANG: en_CA.utf8
LOCALE: en_CA.UTF-8

pandas: 0.20.2
pytest: 2.9.1
pip: 9.0.1
setuptools: 36.2.7
Cython: None
numpy: 1.13.0
scipy: 0.19.0
xarray: None
IPython: 5.1.0
sphinx: None
patsy: None
dateutil: 2.5.3
pytz: 2016.4
blosc: None
bottleneck: None
tables: None
numexpr: None
feather: None
matplotlib: 2.0.2
openpyxl: None
xlrd: None
xlwt: None
xlsxwriter: None
lxml: None
bs4: None
html5lib: 1.0b10
sqlalchemy: None
pymysql: None
psycopg2: None
jinja2: 2.8
s3fs: None
pandas_gbq: None
pandas_datareader: None

</details>

### PR DESCRIPTION
**Pull request by ghasemnaddaf**
- [ ] closes #18206
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/indexes/multi.py [hunk 1]
```diff
@@ -811,7 +811,9 @@ def duplicated(self, keep='first'):
 
     @Appender(ibase._index_shared_docs['fillna'])
```
Comment (ghasemnaddaf) [inline pandas/core/indexes/multi.py]: should this line be removed?

Comment (gfyoung) [reply pandas/core/indexes/multi.py]: I don't think so.  That appends additional documentation.

Comment (gfyoung) [reply pandas/core/indexes/multi.py]: Ah, but I see #18018.  In that case, leave this alone for this PR.  Make the change in a subsequent PR.

File: pandas/core/indexes/multi.py [hunk 2]
```diff
@@ -811,7 +811,9 @@ def duplicated(self, keep='first'):
...
     def fillna(self, value=None, downcast=None):
-        # isna is not implemented for MultiIndex
+        """
+        isna is not implemented for MultiIndex
```
Comment (jorisvandenbossche) [inline pandas/core/indexes/multi.py]: isna -> fillna

File: pandas/core/series.py [hunk 3]
```diff
@@ -1336,7 +1336,8 @@ def idxmax(self, axis=None, skipna=True, *args, **kwargs):
         Parameters
         ----------
         skipna : boolean, default True
-            Exclude NA/null values
+            Exclude NA/null values. If an entire row/column is NA, the result
```
Comment (jorisvandenbossche) [inline pandas/core/series.py]: "an entire row/column" -> "the entire Series"

**Comment by jorisvandenbossche** [comment]
@ghasemnaddaf Thanks! Could you also add a 'Raises' section in the idxmin/idxmax docstring to say that it raises a ValueError when for empty rows/columns ?

File: pandas/core/indexes/multi.py (hunk 2 above)
Comment (ghasemnaddaf) [reply pandas/core/indexes/multi.py]: @jorisvandenbossche please  see PR #18269 for changes related  to `fillna`.

File: pandas/core/indexes/multi.py (hunk 1 above)
Comment (ghasemnaddaf) [reply pandas/core/indexes/multi.py]: Moved `fillna` changes to PR #18269 .

**Comment by ghasemnaddaf** [comment]
added `Raises` section

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@ghasemnaddaf Thanks!

**Pull request merged**